docs/src/pages/start/pick-quasar-flavour.md

```diff
@@ -21,22 +19,31 @@ Here's a quick comparison:
 | Progressive integration of Quasar                                                          | **Yes** | -          | -                             |
 | Include Quasar from public CDN                                                             | **Yes** | -          | -                             |
 | Build SPA, PWA                                                                             | **Yes** | **Yes**    | **Yes**                       |
-| Build SSR (+ optional PWA client takeover)                                                 | -       | **Yes**    | Yes*                          |
-| Build Mobile Apps, Electron Apps                                                           | **Yes** | **Yes**    | Yes*                          |
-| Build Browser Extensions                                                            | -       | **Yes**    | Yes*                          |
+| Build SSR (+ optional PWA client takeover)                                                 | -       | **Yes**    | Yes(*)                          |
+| Build Mobile Apps via Cordova or Capacitor                                                 | **Yes** | **Yes**    | Yes(*)                          |
+| Develop Mobile Apps with HMR directly on your phone.                                       | -       | **Yes**    | Yes(*) |
+| Build Desktop Apps via Electron                                                            | -       | **Yes**    | Yes(*)                             |
+| Build Browser Extensions                                                                   | -       | **Yes**    | Yes(*)                          |
 | Quasar **App Extensions**                                                                  | -       | **Yes**    | - |
-| App icons & splash screens easy management through [Icon Genie](/icongenie/introduction) | - | **Yes** | - |
+| Easy management of App icons & splash screens via [Icon Genie](/icongenie/introduction)    | - | **Yes** | - |
```

Comment:
I suggest making this "Icon Genie Cli". Too many people getting mixed up with older AE version.

Reply:
I agree but with CLI in caps :)

Reply:
Done.